src/main/scala/com/raquo/laminar/shoelace/sl/CommonTypes.scala

```diff
@@ -9,6 +9,7 @@ import com.raquo.laminar.keys.DerivedStyleProp
 import com.raquo.laminar.modifiers.KeySetter
 import com.raquo.laminar.modifiers.KeySetter.StyleSetter
 import org.scalajs.dom
+import scala.scalajs.{js as sjs}
```

Comment:
If you just imported as `js`, would that cause a collision? I think it is more common in Scala.js projects to import the `js` as is.